addons/web_editor/controllers/main.py

```diff
@@ -310,14 +310,29 @@ def get_assets_editor_resources(self, key, get_views=True, get_scss=True, bundle
                         if excluded_url_matcher.match(url):
                             continue
 
+                        # Set files URL filters
+                        custom_css_files = [
+                            '/website/static/src/scss/bootstrap_variables_override.scss',
+                            '/website/static/src/scss/css_custom_rules.scss',
+                            '/website/static/src/scss/css_custom_variables.scss']
+                        url_from_custom_files = True if url in custom_css_files else False
+                        url_from_website_files = url.startswith('/website')
+
                         # Check if the file is customized and get bundle/path info
                         scss_file_data = self._match_scss_file_url(url)
                         if not scss_file_data:
                             continue
 
-                        # Save info (arch will be fetched later)
+                        # Save info according to the filter (arch will be fetched later)
                         url_infos[url] = scss_file_data
-                        scss_files_data.append(url)
+                        if files_filter == 'default':
+                            if url_from_custom_files:
+                                scss_files_data.append(url)
+                        elif files_filter == 'all_website':
```

Comment:
The 'website' filter is not for the files which starts with '/website' it is the default filter of the current master, which is simply using the bundle_restriction.